build.sbt

```diff
@@ -116,6 +116,7 @@ lazy val dbeamCore = project
       "org.apache.commons" % "commons-dbcp2" % "2.1.1",
       "org.postgresql" % "postgresql" % "42.2.+",
       "mysql" % "mysql-connector-java" % "5.1.+",
+      "com.microsoft.sqlserver" % "mssql-jdbc" % "6.4.0.jre8",
```

Comment:
I would prefer not to add `mssql-jdbc` jar as a dependency to `dbeam-core`. MySQL and PostgreSQL are defined as dependencies because of the GA support from Google Cloud SQL.

That being said, I don't think it is a problem to add some code to handle sqlserver. I suggest as an alternative to create a `dbeam-mssql` project/artifact on `build.sbt` which would depend on `mssql-jdbc`.